exodus_core/analysis/static_analysis.py

```diff
@@ -325,9 +325,9 @@ def _get_icon_from_details(self, path):
         if details is not None:
             for i in details.get('images'):
                 if i.get('imageType') == 4:
-                    f = urllib.request.urlopen(i.get('url'))
+                    f = requests.get(i.get('url'))
                     with open(path, mode='wb') as fp:
-                        fp.write(f.read())
+                        fp.write(f.text)
```

Comment:
this should be f.content, otherwise it will raise an error I think

Reply:
You are right. 

.text returns text and .content returns a bytes response. Your choice is more pertinent while handling a file in binary mode. I am going to change that. According to this link https://stackoverflow.com/questions/17011357/what-is-the-difference-between-content-and-text

Reply:
.text returns a string and here we need a bytes response indeed. So simply replacing this with f.content will do the trick :)